docs/gambit/mutation-verifier.md

```diff
@@ -272,11 +269,39 @@ At the top bar of the report there are different coverage metrics.
 
 ![Mutation metrics](doc/mutation_metrics.png)
 
-- The Coverage metric is the ratio of the caught mutants to all mutants tested, 
-  also shown under Caught Mutations.
-- The Rules metric shows the ratio of the rules that caught at least one mutation 
+- The _Coverage_ metric is the ratio of the caught mutants to all mutants tested, 
+  also shown under _Caught Mutations_.
+- The _Rules_ metric shows the ratio of the rules that caught at least one mutation 
   out of all the rules in the tested specification.
-- The Solo Rules metric shows the ratio between the rules that caught a unique mutation 
+- The _Solo Rules_ metric shows the ratio between the rules that caught a unique mutation 
   and all rules that caught at least one mutation.
 
+## CLI Options
+
+`certoraMutate` supports the following options; for a comprehensive list, run `certoraMutate --help`:
+
+| <div style="width:160px">Option</div> | Description                                                                                                           |
+|:--------------------------------------|:----------------------------------------------------------------------------------------------------------------------|
+| `--conf`                     | Specify the path to the {ref}`configuration <mut-conf>` file                                                                 |
+| `--orig_run`                          | Specify the {ref}`link to a previous verification job<orig-verification-link>`                                                            |
+| `--orig_run_dir`                      | Specify the folder where the files will be downloaded from the {ref}`original verification job <orig-verification-link>` |
+| `--msg`                               | Add a message to identify the `certoraMutate` run |
+| `--gambit_only`                       | Stop processing after generating mutations |
+| `--dump_failed_collects`              | Specify a log file to capture mutant collection failures |
+| `--debug`                             | Show additional logging information during execution                                                                  |
 
+## Troubleshooting
+
+At the moment, there are a few ways in which `certoraMutate` can fail. 
+Here are some suggestions on how to troubleshoot when that happens. 
+We are actively working on mitigating them.
+
+- There are currently no official Gambit binaries for Linux ARM or Windows. That means that Gambit will not be installed with `pip install certora-cli`, and must be {ref}`built from source <build-gambit-from-source>`.
+
```

Comment:
should we also add a bullet here to remove trailing `/` from packages?

Reply:
done